src/main/java/kitchenpos/table/application/TableService.java

```diff
@@ -0,0 +1,41 @@
+package kitchenpos.table.application;
+
+import java.util.*;
+import java.util.stream.*;
+
+import org.springframework.stereotype.*;
+
+import kitchenpos.common.*;
+import kitchenpos.table.domain.*;
+import kitchenpos.table.dto.*;
+import kitchenpos.table.repository.*;
+
+@Service
+public class TableService {
+    private static final String ORDER_TABLE = "주문 테이블";
+    private final OrderTableRepository orderTableRepository;
```

Comment:
```suggestion
    private static final String ORDER_TABLE = "주문 테이블";
    
    private final OrderTableRepository orderTableRepository;
```

클래스 변수와 인스턴스 변수 사이에 공백을 추가하여 가독성을 높여보아요!

Reply:
넵, 알겠습니다.